client/layout/guided-tours/tours/jetpack-lazy-images-tour/index.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * External dependencies
+ */
+import React, { Fragment } from 'react';
+import Gridicon from 'gridicons';
+
+/**
+ * Internal dependencies
+ */
+import meta from './meta';
+import {
+	ButtonRow,
+	Continue,
+	makeTour,
+	Quit,
+	SiteLink,
+	Step,
+	Tour,
+} from 'layout/guided-tours/config-elements';
+
+/* eslint-disable wpcalypso/jsx-classname-namespace */
+export const JetpackLazyImagesTour = makeTour(
+	<Tour { ...meta }>
+		<Step
+			name="init"
+			target=".jetpack-lazy-images-settings .form-toggle__switch"
+			arrow="top-left"
+			placement="below"
+			style={ {
+				animationDelay: '0.7s',
+				zIndex: 1,
+			} }
+		>
+			{ ( { translate } ) => (
+				<Fragment>
+					<p>{ translate( "Let's boost load times for your site by lazy-loading images." ) }</p>
+					<ButtonRow>
+						<Continue
+							target=".jetpack-lazy-images-settings .form-toggle__switch"
+							step="finish"
+							click
+							hidden
+						/>
+						<SiteLink href="/plans/my-plan/:site">
+							{ translate( 'Return to the checklist' ) }
+						</SiteLink>
+					</ButtonRow>
+				</Fragment>
+			) }
+		</Step>
+
+		<Step name="finish" placement="right">
+			{ ( { translate } ) => (
+				<Fragment>
+					<h1 className="tours__title">
+						<span className="tours__completed-icon-wrapper">
+							<Gridicon icon="checkmark" className="tours__completed-icon" />
+						</span>
+						{ translate( 'Excellent, you’re done!' ) }
+					</h1>
+					<p>
+						{ translate(
+							'Lazy image loading has been enabled. Would you like to continue setting up performance features for your site?'
```

Comment:
Lazy-loading images are active on your site. Ready to set up some more performance-improving features?